src/databricks/labs/dqx/engine.py

```diff
@@ -999,6 +999,21 @@ def load_checks_from_table(self, table_name: str, run_config_name: str = "defaul
     def save_checks_in_local_file(checks: list[dict], path: str):
         return DQEngineCore.save_checks_in_local_file(checks, path)
 
+    @staticmethod
+    def convert_quality_rules_to_metadata(checks: list[DQRule]) -> list[dict]:
```

Comment:
To follow naming convention of other similar methods, should this be called `convert_checks_to_metadata`